packages/core/src/app/ui/loading/LoadingSpinnerOverlay.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FunctionComponent } from 'react';
+
+interface LoadingSpinnerOverlayProps {
+    isLoading: boolean;
+    baseColor?: string;
+    spinnerColor?: string;
+    overlayColor?: string;
+}
+
+const SPINNER_ANIMATION = 'embedded-checkout-loading-spinner-overlay-rotation';
+
+const LoadingSpinnerOverlay: FunctionComponent<LoadingSpinnerOverlayProps> = ({
```

Comment:
Is there a reason we are creating a new indicator? Can we not use the existing loading indicator already present in checkout js? We should try and keep one style of loading indicator to keep a uniform design.